submissions/mustbefail/oop/index.js

```diff
@@ -0,0 +1,86 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+  Complete the below for code reviewers' convenience:
+
+  Code repository: _put repo URL here_
+  Web app: _put project's github pages URL here_
+  */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+  constructor(name, gender, phrase, friends, legs) {
+    this.friends = friends ? friends : ['no friends'];
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.phrase = phrase;
+  }
+  saying() {
```

Comment:
Why do you need this method?

Reply:
I use logic: an action is a function

Reply:
Logic is good. Then function name should be a verb or start with a verb in imperative mood. E.g. `getProperties`, `sendMessage`, `jump`, `say`.
You are not required to fix this. Fix if you want use this project as a part of your portfolio to show to potential employers.

Reply:
Fixed